srv/isc-agent/README.MD

```diff
@@ -0,0 +1,63 @@
+# README #
+
+## What is this repository for? ##
+
+[TODO: add description]
+
+## How do I get set up? ##
+
+1. Install [docker](https://docs.docker.com/get-docker/) by following the steps in the link provided.
+2. Clone the repository to your computer.
+    ```shell script
+    git clone git@github.com:DShield-ISC/dshield.git {{ project location }}
+    ``` 
+3. Go to the project directory and change the branch to `epic/ISC-1-web-rebuild`
+    ```shell script
+    cd {{ project location }}
+    git checkout epic/ISC-1-web-rebuild
+    git pull
+    ``` 
+4. Run the following command to start the project.
+    ```shell script
+    # This command should be ran anytime new packages are added to the project.
+    # It is good habit to rebuild everytime you pull down new code.
+    make docker-up
```

Comment:
Have to cd to folder containing Makefile. Not sure if this needs to be added to the README.md